client/hotrod-client/src/test/java/org/infinispan/client/hotrod/configuration/ConfigurationTest.java

```diff
@@ -155,30 +270,52 @@ private void assertServer(String host, int port, ServerConfiguration serverCfg)
 
    private void validateConfiguration(Configuration configuration) {
       assertEquals(2, configuration.servers().size());
-      assertEquals(SomeAsyncExecutorFactory.class, configuration.asyncExecutorFactory().factoryClass());
-      assertEquals(SomeRequestBalancingStrategy.class, configuration.balancingStrategyClass());
-      assertEquals(SomeTransportfactory.class, configuration.transportFactory());
+      assertEqualsConfig(SomeAsyncExecutorFactory.class, ASYNC_EXECUTOR_FACTORY, configuration);
```

Comment:
The idea behind it is to be able to reuse them potentially in other client config tests.